lib/ngtsc/symbol.ts

```diff
@@ -112,14 +110,4 @@ export abstract class Symbol<A extends AnnotationNames> {
       this.analyse();
     }
   }
```

Comment:
As we remove this feature, it might be good to explain how to achieve the same behavior in the readme.